src/sw/redis++/connection_pool.cpp

```diff
@@ -55,13 +55,13 @@ ConnectionPool::ConnectionPool(SimpleSentinel sentinel,
     assert(_sentinel);
 }
 
-ConnectionPool::ConnectionPool(ConnectionPool &&that) {
+ConnectionPool::ConnectionPool(ConnectionPool &&that) noexcept {
```

Comment:
Both `std:::lock_guard` and `std::lock` do not have `noexcept` specifier. I don't think it's a good idea to add `noexcept` for `ConnectionPool`'s constructor and assignment operator.

Reply:
Ah, good point and I don't have a clean solution other than catching any `system_error` and `std::terminating` :-) Removing that part.